includes/forms/class-message-report.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Message report form.
+ *
+ * @package HivePress\Forms
+ */
+
+namespace HivePress\Forms;
+
+use HivePress\Helpers as hp;
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Reports message.
+ */
+class Message_Report extends Model_Form {
```

Comment:
Is this form was copied from another one, e.g. listing report form? I don't remember exactly but the Model_Form inheritance may not be needed in this case and may cause conflicts, but maybe it has a purpose